assets/cubyz/biomes/glacier/base.zig.zon

```diff
@@ -0,0 +1,90 @@
+.{
+	.properties = .{
+		.cold,
+		.wet,
+		.barren,
+		.ocean,
+	},
+	.minHeight = -64,
+	.maxHeight = 180,
+	.minRadius = 130,
+	.maxRadius = 256,
+	.roughness = 30,
+	.mountains = 50,
+	.hills = 50,
+	.ground_structure = .{
+		"3 to 5 cubyz:snow",
+		"4 cubyz:frost",
+	},
+	.stoneBlock = "cubyz:frost",
+	.structures = .{
+		.{
+			.id = "cubyz:stalagmite",
+			.block = "cubyz:snow",
+			.chance = 0.0075,
+			.size = 7,
+			.size_variation = 6,
+			.generationMode = .water_surface,
+		},
+		.{
+			.id = "cubyz:stalagmite",
+			.block = "cubyz:frost",
+			.chance = 0.025,
+			.size = 4,
+			.size_variation = 5,
+			.generationMode = .water_surface,
+		},
+		.{
+			.id = "cubyz:stalagmite",
+			.block = "cubyz:ice",
+			.chance = 0.04,
+			.size = 1,
+			.size_variation = 4,
+			.generationMode = .water_surface,
+		},
+		.{
+			.id = "cubyz:ground_patch",
+			.block = "cubyz:ice",
+			.chance = 0.025,
+			.width = 6,
+			.variation = 18,
+			.depth = 3,
+			.smoothness = 0.5,
+			.generationMode = .water_surface,
+		},
+		.{
+			.id = "cubyz:ground_patch",
+			.block = "cubyz:ice",
+			.chance = 0.125,
+			.width = 3,
+			.variation = 18,
+			.depth = 2,
+			.smoothness = 0.66,
+			.generationMode = .water_surface,
+		},
+		.{
+			.id = "cubyz:ground_patch",
+			.block = "cubyz:ice",
+			.chance = 0.3,
+			.width = 1,
+			.variation = 10,
+			.depth = 1,
+			.smoothness = 0.66,
+			.generationMode = .water_surface,
+		},
+	},
+	.transitionBiomes = .{
+		.{
+			.id = "cubyz:beach",
+			.chance = 1,
+			.width = 1,
+			.properties = .{.land, .inland},
+		},
+		.{
+			.id = "cubyz:glacier/floor",
+			.chance = 1,
+			.width = 2,
+			.properties = .{.land, .inland},
+		},
```

Comment:
This seems to have uninentional consequences:
![Screenshot at 2025-03-26 17-29-20](https://github.com/user-attachments/assets/497086ee-4a19-497b-a472-dad702b62c14)